### ISSUE TITLE
Keyboard navigation does not work with layouts other than English

### ISSUE THREAD

**Issue by lue** [mozilla/hubs#2330]
**Description**
If my keyboard layout is switched to Russian, WASD keys do not work. It might be the issue with other layouts too.

**To Reproduce**
Switch to Russian keyboard layout. Try to navigate with WASD.

**Expected behavior**
Independent of the keyboard layout WASD buttons should always work.

**Hardware** 
 - Desktop
 - OS: Windows 10
 - Chrome, Edge

**Comment by vincentfretin** [comment]
duplicate of #2397

**Comment by misslivirose** [comment]
Resolving as duplicate of #2397 - thanks for reporting this!
